src/common/board.rs

```diff
@@ -1,24 +1,22 @@
+//! Abstraction of a general-purpose compute board
+
 #![allow(dead_code)]
-use crate::common::analog::AnalogReader;
-use crate::common::status::Status;
-use crate::google;
-use crate::proto::common;
-use crate::proto::component;
+use crate::{
+    common::{analog::AnalogReader, status::Status},
+    google,
+    proto::{common, component::board::v1::PowerMode},
```

Comment:
nit: would just import proto::component so we know PowerMode is a generated type